common/c/targets/posix/target_gpio_led.c

```diff
@@ -0,0 +1,51 @@
+/*
+ *  Copyright 2014-2016 CyberVision, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+#include <stdint.h>
+#include <stdio.h>
+
+#include "target.h"
+#include "target_gpio_led.h"
+
+static bool gpio_led_status[NUM_GPIO_LEDS];
+
+void target_gpio_led_init(void)
+{
+    int n;
+    
+    for (n = 0; n < NUM_GPIO_LEDS; n++) {
+        gpio_led_status[n] = false;
```

Comment:
`gpio_led_status` is a static variable and is initialized with zeros by default.